api_proxy.yaml

```diff
@@ -676,9 +676,21 @@ definitions:
       - https://www.spid.gov.it/SpidL2
       - https://www.spid.gov.it/SpidL3
   UserProfile:
-    oneOf:
```

Comment:
it seems the previous version was correct, what was the error?

Reply:
The `oval` linter returns this error (here the [link](https://circleci.com/gh/teamdigitale/io-backend/1682) at the build)
```#/definitions/UserProfile: Additional properties not allowed: oneOf```
According with the [documentation](https://swagger.io/docs/specification/data-models/oneof-anyof-allof-not/) examples seems that the keywords `oneOf` and `anyOf` are usable only inside response schema definition and only `allOf` is allowed in Type schema definition.
Maybe i'm missing something.

Reply:
that's right, I think these specs needs more fixes, I'll deal with them in #415

Reply:
@BurnedMarshal this is now fixed on master, see https://github.com/teamdigitale/io-backend/blob/master/api_backend.yaml#L661